cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc

```diff
@@ -0,0 +1,17 @@
+[id="cnv-monitoring-vm-health"]
+= Monitoring virtual machine health
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-monitoring-vm-health
+toc::[]
+
+Liveness probes stop the `VirtualMachineInstance` if it fails and allow
+controllers such as `VirtualMachine` or `VirtualMachineInstanceReplicaSet` to spawn
+other `VirtualMachineInstances`, in order to restore responsiveness to the
+virtual machine.
+
+Readiness probes are indicators for Services and Endpoints that
```

Comment:
- If this is the first place we're discussing readiness probes, I think that the term should be italicized (as with liveness probes)
- Why are Services and Endpoints capitalized?

Reply:
Yes, it is first occurrence so I italicized.
Fixed capitals.